src/main/java/gregtech/api/capability/impl/MultiblockRecipeLogic.java

```diff
@@ -103,6 +104,43 @@ protected IMultipleTankHandler getOutputTank() {
         return controller.getOutputFluidInventory();
     }
 
+    @Override
+    protected boolean canWorkWithInputs() {
+        MultiblockWithDisplayBase controller = (MultiblockWithDisplayBase) metaTileEntity;
+        if (controller instanceof RecipeMapMultiblockController) {
+            RecipeMapMultiblockController distinctController = (RecipeMapMultiblockController) controller;
+
+            if (distinctController.canBeDistinct() && distinctController.isDistinct()) {
+                boolean canWork = false;
+                if (!hasNotifiedInputs() && !hasNotifiedOutputs()) {
```

Comment:
Since this is in `canWorkWithInputs`, checking if the outputs are notified is probably not correct. That is done in `canFitNewOutputs`.